frontend/src/metabase/admin/databases/containers/DatabaseEditApp.jsx

```diff
@@ -17,15 +18,14 @@ import * as databaseActions from "../database";
 
 const mapStateToProps = (state, props) => {
     return {
-        databaseId:       state.router && state.router.params && state.router.params.databaseId,
+        databaseId:       props.params.databaseId,
         database:         getEditingDatabase(state),
-        formState:        getFormState(state),
-        onChangeLocation: props.onChangeLocation
+        formState:        getFormState(state)
     }
 }
 
 const mapDispatchToProps = {
-    ...databaseActions
+    ...databaseActions,
```

Comment:
Yeah it's pretty inconsistent all over the app, so don't worry about this stuff for now unless we've already established a preference.